deploy/deployment-manager/service_account.py

```diff
@@ -0,0 +1,53 @@
+"""A template file for deploying a service account for the elastic agent."""
+
+
+# pylint:disable=duplicate-code
```

Comment:
A code duplication because we need to pass `properties` dynamically through the command line and only templates support it, however, templates cannot import modules 😢, therefore. I cannot create a common module and import it into different templates.